SBLGNT/lowfat/05-acts.xml

```diff
@@ -291543,7 +291543,7 @@
                         case="dative"
                         english="council"
                         mandarin="议会"
-                        gloss="he said"
+                        gloss="Council"
```

Comment:
This document and other SBLGNT documents contain many ambiguous Unicode characters.